### PR TITLE
feat(admin): support `ruder admin:init` and `ruder admin:info`

### LINKED ISSUE #12: [feature]Support `rudr admin:init` which will check/install oam-k8s-runtime into cluster
1. check K8s Cluster avaliable. (kubeconfig)
2. check oam-k8s-runtime is avaliable in cluster, or install it by helm.
3. create workloadDefinition/traitDefinition for K8s built-in resource(Deployment,StatefulSet, etc)

### PR DESCRIPTION
**Pull request by Ghostbaby**
Support `ruder admin:init` and `ruder admin:info`

Fixes #12

Signed-off-by: zhuhuijun <zhuhuijunzhj@gmail.com>

### CODE REVIEW

File: pkg/cmd/init.go [hunk 1]
```diff
@@ -75,12 +81,44 @@ var (
...
+		},
+	}
+
+	return cmd
+}
+
+func (i *infoCmd) run(version string, ioStreams cmdutil.IOStreams) error {
+	clusterVersion, err := GetOAMReleaseVersion()
+	if err != nil {
+		ioStreams.Errorf("fail to get cluster version")
```
Comment (wonderflow) [inline pkg/cmd/init.go:101]: forget return error message?

File: pkg/cmd/init.go [hunk 2]
```diff
@@ -75,12 +81,44 @@ var (
...
+	}
+
+	if len(clusterVersion) < 1 {
+		ioStreams.Info("oam-kubernetes-runtime not found in your kubernetes cluster, please use `ruder admin:init` to install.")
+		ioStreams.Infof("client  version: %s \n", version)
+		return nil
+	}
```
Comment (wonderflow) [inline pkg/cmd/init.go:108]: These tips should print in error check above.

File: pkg/cmd/init.go [hunk 3]
```diff
@@ -170,7 +208,7 @@ func NewHelmInstall(version string, ioStreams cmdutil.IOStreams) (*action.Instal
 		kube.GetConfig(cmdutil.GetKubeConfig(), "", types.DefaultOAMNS),
 		types.DefaultOAMNS,
 		os.Getenv("HELM_DRIVER"),
-		ioStreams.Infof,
+		info,
```
Comment (wonderflow) [inline pkg/cmd/init.go:211]: why need this change?

File: pkg/cmd/init.go [hunk 4]
```diff
@@ -251,6 +293,35 @@ func GetHelmRepositoryList() []*repo.Entry {
 	return nil
 }
 
+func GetHelmRelease() ([]*release.Release, error) {
+	actionConfig := new(action.Configuration)
+	client := action.NewList(actionConfig)
+
+	if err := actionConfig.Init(settings.RESTClientGetter(), "", os.Getenv("HELM_DRIVER"), info); err != nil {
```
Comment (wonderflow) [inline pkg/cmd/init.go:300]: what will happen if `os.Getenv("HELM_DRIVER")` is empty?

File: pkg/cmd/init.go [hunk 5]
```diff
@@ -251,6 +293,35 @@ func GetHelmRepositoryList() []*repo.Entry {
...
+	if err != nil {
+		return "", err
+	}
+
+	for _, result := range results {
+		if result.Chart.ChartFullPath() == types.DefaultOAMRuntimeName {
+			return result.Chart.AppVersion(), nil
+		}
+	}
+	return "", nil
```
Comment (wonderflow) [inline pkg/cmd/init.go:322]: we could return a `not exist` erorr here.

File: pkg/cmd/init.go (hunk 3 above)
Comment (Ghostbaby) [reply pkg/cmd/init.go:211]: this is a log debug switch, so we need to input a debug log function.

File: pkg/cmd/init.go (hunk 4 above)
Comment (Ghostbaby) [reply pkg/cmd/init.go:300]: Init will create a new secret in k8s to store release info.

**Comment by zzxwill** [comment]
@Ghostbaby Please help fix the issue.
```
➜ go run cmd/rudrx/main.go admin:init
Successfully initialized.
➜  go run cmd/rudrx/main.go admin:info
oam-kubernetes-runtime not found in your kubernetes cluster, please use `ruder admin:init` to install.
client  version: UNKNOWN
```

**Comment by Ghostbaby** [comment]
@zzxwill copy

**Pull request merged**